roxctl/generate/netpol/cta.go

```diff
@@ -12,12 +17,82 @@ func (cmd *generateNetpolCommand) generateNetpol() error {
 		return errors.Wrap(err, "Error synthesizing policies from folder")
 	}
 
+	var yamlPolicies []string
+	var mergedPolicy string
 	for _, netpol := range recommendedNetpols {
 		yamlPolicy, err := networkpolicy.KubernetesNetworkPolicyWrap{NetworkPolicy: netpol}.ToYaml()
 		if err != nil {
-			return errors.Wrap(err, "Error converting YAML into Network Policy")
+			return errors.Wrap(err, "Error converting Network Policy object to YAML")
+		}
+		yamlPolicies = append(yamlPolicies, yamlPolicy)
+		mergedPolicy = fmt.Sprintf("%s---\n%s", mergedPolicy, yamlPolicy)
+	}
+
+	if !cmd.mergePolicies && !cmd.splitPolicies {
+		cmd.printNetpolsToStdout(mergedPolicy)
+		return nil
+	}
+
+	if cmd.mergePolicies {
+		if err := cmd.saveNetpolsToMergedFile(mergedPolicy); err != nil {
+			return errors.Wrapf(err, "Error writing merged Network Policies to file")
+		}
+	}
+
+	if cmd.splitPolicies {
+		if err := cmd.saveNetpolsToFolder(recommendedNetpols); err != nil {
+			return errors.Wrapf(err, "Error writing split Network Policies to folder")
 		}
-		cmd.env.Logger().PrintfLn("---\n\n%s", yamlPolicy)
 	}
+
 	return nil
 }
+
+func (cmd *generateNetpolCommand) printNetpolsToStdout(combinedNetpols string) {
+	cmd.env.Logger().PrintfLn(combinedNetpols)
```

Comment:
I hope this logger cannot be disabled with some extreme log-level setting

Reply:
True. I based this on the Use Case that a User actually wants to see the output, so I wouldn't hope they crank the loglevel to something extreme. 
That said, is the roxctl loglevel configurable through an env variable? I don't think I've seen something like that so far.

Reply:
I think this function should actually print to stdout. This is not a logging message that is printed, that's the output.